physionet-django/project/pbmigration.py

```diff
@@ -0,0 +1,75 @@
+import datetime
+import os
+import pdb
+import pytz
+import requests
+from urllib.request import urlopen
+
+from django.core import serializers
+
+from project.models import LegacyProject
+
+
+def load_legacy_html(slug):
+    """
+    Load data from a pbank database and create a LegacyProject object
+    # Some pages end with this: <!--#include virtual="/dir-footer.shtml"-->
+    """
+    try:
+        f = urlopen('https://physionet.org/physiobank/database/{}/HEADER.shtml'.format(slug))
+    except:
+        f = urlopen('https://physionet.org/physiobank/database/{}/index.shtml'.format(slug))
+    content = f.read().decode('utf8')
+    f.close()
```

Comment:
Seems odd that all this is sitting in the module. Is this intended? Might be worth moving to functions and adding a main() if it is intended to be run as a script? Will help to avoid strange things happening on import.

Reply:
This module isn't imported or called from anywhere else. I just use its snippets in the shell when appropriate at various stages of the manual content editing.